### ISSUE TITLE
howto change keyboard layout?

### ISSUE THREAD

**Issue by 3x3cut0r** [jlesage/docker-firefox#175]
hi, is there a way to change the keyboard layout to german to be able to write special characters like 'äüöÄÜÖ'

**Comment by jlesage** [comment]
Not sure I understand the question/problem.  You should be able to do it on host itself ?

**Comment by 3x3cut0r** [comment]
so you mean the keyboard layout is based on the host kb layout?
what is if i want another layout inside the container as on the host?

**Comment by jlesage** [comment]
Exactly.  The VNC protocol takes care of that.  A keyboard layout is not really used inside the container.

**Comment by 3x3cut0r** [comment]
can i somehow change the keyboard layout inside a docker container?
for example in the linuxserver/firefox container there is a possibility with a "KEYBOARD"-environment variable to change the layout. could you also implement this?

**Comment by jlesage** [comment]
linuxserver/firefox is using RDP instead of VNC.  This is why a keyboard layout make sense in that case.

Keyboard layout doesn't apply with VNC.  So I don't know how this could be done...  Why using the host's keyboard layout doesn't work for you ?

**Comment by 3x3cut0r** [comment]
unfortunetaly not :/
host and guest need to be different
